### ISSUE TITLE
Question: Is OX-Coi starting to die?

### ISSUE THREAD

**Issue by webratte** [open-xchange/ox-talk#619]
Hi,
I got just a question.
Your last commit is now 1 month ago.

What happens?
I would like to see a second Mail-messenger side by side to Delta Chat.

**Comment by robert-virkus** [comment]
Hi, we are reorganizing things, I hope I can announce some good news soon.

**Comment by linuxmalaysia** [comment]
Yes please. Work with Delta Chat to build better E2EE chat system.

**Comment by helmut72** [comment]
Sorry, didn't read the readme.
